File: src/OnPeople/BackEnd/src/OnPeople.Domain/Models/Empresas/Empresa.cs
using OnPeople.Domain.Models.Departamentos;
using OnPeople.Domain.Models.Users;

namespace OnPeople.Domain.Models.Empresas
{
    public class Empresa
    {
        public int Id { get; set; }  
        public string Cnpj { get; set; }
        public string RazaoSocial { get; set; }
        public string PorteEmpresa { get; set; }
        public string NaturezaJuridica { get; set; }
        public string OptanteSimples { get; set; }
        public Boolean  Filial { get; set; }
        public string NomeFantasia { get; set; }
        public Boolean Ativa { get; set; }
        public string SiglaEmpresa { get; set; }
        public string DataCadastro { get; set; }
        public string DataDesativacao { get; set; }
        public string PadraoEmail { get; set; }
        public int MatrizId { get; set; }
        public string Logotipo { get; set; }
        public int PresidenteId {get; set;}
        public string TipoLogradouro { get; set; }
        public string Logradouro { get; set; }
        public string Numero { get; set; }
        public string Complemento { get; set; } 
        public string Bairro { get; set; }
        public string CEP { get; set; }
        public string DDD { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string AtividadePrincipal { get; set; }
        public string PaisId { get; set; }
        public string SiglaPaisIso2 { get; set; }
        public string SiglaPaisIso3 { get; set; }
        public string NomePais { get; set; }
        public int EstadoId { get; set; }
        public string Estado { get; set; }
        public string SiglaEstado { get; set; }
        public int EstadoIbgeId { get; set; }
        public int CidadeId { get; set; }
        public string Cidade { get; set; }
        public int CidadeIbgeId { get; set; }
        public string CidadeSiafiId { get; set; }
        public IEnumerable<User> Users { get; set; }
        public IEnumerable<Departamento> Departamentos { get; set; }
    }
}

File: src/OnPeople/FrontEnd/OnPeople-App/src/app/companies/models/Empresa.ts
import { EmpresaConta } from "./EmpresaConta";


export interface Empresa {
  id: number;
  cnpj: string;
  razaoSocial: string;
  nomeFantasia: string;
  siglaEmpresa: string;
  ativa: Boolean;
  dataCadastro: string;
  dataDesativacao: string;
  filial: Boolean;
  padraoEmail: string;
  presidenteId: number;
  naturezaJuridica: string;
  porteEmpresa: string;
  optanteSimples: string;
  tipoLogradouro: string;
  logradouro: string;
  numero: string;
  complemento: string;
  bairro: string;
  cep: string;
  ddd: string;
  telefone: string;
  email: string;
  atividadePrincipal: string;
  siglaPaisIso3: string;
  siglaPaisIso2: string;
  nomePais: string;

  matrizId: number;
  logotipo: string;
  paisId: string;
  estadoId: number;
  estado: string;
  siglaEstado: string;
  estadoIbgeId: number;
  cidadeId: number;
  cidade: string;
  cidadeIbgeId: number;
  cidadeSiafiId: string;
  empresasContas: EmpresaConta[];
}
